### PR TITLE
[REF] auto-fix conventions

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

File: connector_ecommerce/sale.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def _get_payment_default(self, cr, uid, context=None):
 
 
 class sale_order(orm.Model):
+
```
Comment (guewen) [inline connector_ecommerce/sale.py:55]: From where does come this idea to put a blank line between the class and the docstring?
This is not what says the pep0257: http://legacy.python.org/dev/peps/pep-0257/

> - There's no blank line either before or after the docstring.
> 
> Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.

**Comment by guewen** [comment]
Disagree because it adds blank lines between the class definition and the docstring. :-1:

**Comment by yvaucher** [comment]
@moylop260 Can you remove the blank lines?

**Comment by yvaucher** [comment]
Fixed in https://github.com/OCA/connector-ecommerce/pull/17

**Pull request closed without merging**